python/tests/rd_tests/test_sum.py

```diff
@@ -1,7 +1,9 @@
 import csv
+import pytest
```

Comment:
One of these imports is a duplicate?

Reply:
resdata linting does not reorder or merge imports as the module structure is quite fragile.